clients/contracts/bridge.ts

```diff
@@ -3,11 +3,9 @@ import { DelphinusContract, DelphinusWeb3 } from "web3subscriber/src/client";
 import { decodeL1address } from "web3subscriber/src/addresses";
 import { PromiseBinder } from "web3subscriber/src/pbinder";
 import { TokenContract } from "./token";
-import { Tokens, Chains } from "./tokenlist";
-import { extraTokens} from "delphinus-deployment/config/extratokens";
-const BridgeContractABI = require("../../build/contracts/Bridge.json");
+import { extraTokens, Chains, contractsInfo } from "delphinus-deployment/config/contractsinfo";
```

Comment:
First, in config-contracts-info.ts, the file name is contracts-info.json.

Reply:
it was in https://github.com/DelphinusLab/delphinus-deployment/commit/c6311ce4d33b84b2efe2fc8df5e4f31dbbd2be81